### ISSUE TITLE
(Nameplates) Casting

### ISSUE THREAD

**Issue by Bzmode** [ccshiro/cc-buglist#686]
there is a delay on the cast bar attached to nameplates but it works fine on player frames
http://www.twitch.tv/faleene/c/6801543

**Comment by cdreipo** [comment]
yes i witnessed that too, it pretty stops on the nameplates which can be confusing.

**Comment by ccshiro** [comment]
Kind of need to figure out if this is a client side bug or something the server does wrong. I think debugging the Lua code would be the most productive approach, but I don't really have the time to spend on that for such a small issue.

The ways I'd go about figuring out if this is client-side:

a) Find a video where it does happen on retail.
Can only prove it's a client-side bug, cannot disprove it, so it's not really an ideal approach. However, anyone can pitch in this way.

b) Look at the Lua code, try to find what the bug is and possibly why it happens.
This requires Lua knowledge, but is probably the better approach.

I realize that it's probably going to be hard to help me with this one, and that it's not a huge bug. But I thought I'd ask for help anyway, just in case someone has the knowledge and time to do it.

**Comment by robinsch** [comment]
@ccshiro Does this always happens or just sometimes?
SMSG_SPELL_START is for both target castbar and nameplate.
For me it looks like that you send SMSG_SPELL_DELAYED and that's somehow breaking the nameplate castbar, because each time the creature takes damage the nameplate castbar makes a drop.

**Comment by Schaka** [comment]
There was a similar bug on retail, where the castbar would just break and stop at ~5%.
Also, you cannot debug the Lua code, because that portion is hardcoded into the client. Pretty much anything related to nameplates is.
